content/post/build-code.md

```diff
@@ -339,7 +339,9 @@ To build LibreOffice, follow the LibreOffice building pages:
 
 https://wiki.documentfoundation.org/Development/BuildingOnLinux
 
-Make sure you use and build the following specific core branch: `distro/collabora/co-23.05` and build with the following config: `./autogen.sh --with-distro=CPLinux-LOKit`
+Make sure you use and build the following specific core branch: `distro/collabora/co-23.05` and build with the following config:
+
+    ./autogen.sh --with-distro=CPLinux-LOKit --without-package-format --without-system-nss
```

Comment:
I'm not sure about `--without-system-nss` I though `--with-distro=CPLinux-LOKit` would already set that (see `distro-configs/CPLinux-LOKit.conf` https://git.libreoffice.org/core/+/refs/heads/distro/collabora/co-23.05/distro-configs/CPLinux-LOKit.conf )

Adding --without-package-format makes sense, thanks.

Reply:
Okay. I've removed it but I'll do a fresh build without --without-system-nss to test this.